Add a section on the admin console partial import/export capability (#27817)
keycloak/keycloak: docs/guides/server/importExport.adoc

In the example above, the value set to the `MY_REALM_NAME` environment variable is going to be used to set the `realm` property.

> == Importing and Exporting via the Admin Console

Reviewer: ```suggestion
== Importing and Exporting by using the Admin Console
```

> It is also possible to import and export a realm using {project_name}'s Admin Console. This functionality is
> different from the other CLI options presented above because the Admin Console offers only the capability to
> *partially* export a realm. In this case, the current realm settings, along with some resources like clients,
> roles, and groups, can be exported. The realm's users *cannot* be exported using this method.

Reviewer: ```suggestion
You can also import and export a realm using the Admin Console. This functionality is
different from the other CLI options described in previous sections because the Admin Console offers only the capability to
_partially_  export a realm. In this case, the current realm settings, along with some resources like clients,
roles, and groups, can be exported. The users for that realm _cannot_ be exported using this method.
```

NOTE: When using the Admin Console export, the realm and the selected resources are always exported to a file
named `realm-export.json`. Also, all sensitive values like passwords and client secrets will be masked with `+*+` symbols.

> To export a realm using the Admin Console, select the realm you want to export, click on *Realm settings*, then
> find the *Action* menu in the top right corner of the realm settings screen, and finally select the *Partial export*
> option. You will get a prompt allowing you to select the resources you want to export along with the realm
> configuration. Check the ones you want to export, and click on *Export*.

Reviewer: ```suggestion
To export a realm using the Admin Console, perform these steps:
. Select a realm.
. Click *Realm settings* in the menu.
. Point to the *Action* menu in the top right corner of the realm settings screen, and select  *Partial export*.
+
A list of resources appears appears along with the realm
configuration. 
. Select the resource you want to export.
. Click *Export*.
```

NOTE: Realms exported from the Admin Console are not suitable for backups or data transfer between servers.
Only CLI exports are suitable for backups or data transfer between servers.

> WARNING: If there is a great number of groups, roles or clients in your realm, the operation may cause the server to be

Reviewer: ```suggestion
WARNING: If there is a great number of groups, roles or clients in your realm, the operation may cause the server to be
```

unresponsive to user requests for a while. Use this feature with caution, especially on a production system.

> In a similar way, you can import a previously exported realm using the same approach. To do this, go to *Realm settings*,
> then find the *Action* menu in the top right corner for the realm settings screen, and finally select the *Partial import*
> option. You will get a prompt that allows you to select the file you want to import, and then, based on the contents
> of this file, you will see the resources you can import along with the realm settings.

Reviewer: ```suggestion
In a similar way, you can import a previously exported realm using the same approach. For this export, perform these steps:
. Click *Realm settings* in the menu.
Point to the *Action* menu in the top right corner of the realm settings screen, and select  *Partial import*.. 
+ A prompt appears where you can select the file you want to import. Based on the contents
of this file, you will see the resources you can import along with the realm settings.
. Click *Import*.
```

> You can also control what {project_name} should do if the imported resource already exists. The options are `Fail import`,
> which aborts the import, `Skip`, which skips the duplicate resources without aborting the process, and `Overwrite`, which
> replaces the existing resources with the ones being imported.

Reviewer: ```suggestion
You can also control what {project_name} should do if the imported resource already exists. These options exist
* `Fail import` - aborts the import.
* `Skip` - skips the duplicate resources without aborting the process
* `Overwrite` - replaces the existing resources with the ones being imported.
```

> NOTE: The Admin Console partial import can also import files created by the `export` CLI command - i.e. full exports created
> via CLI can be imported via Admin Console. If the file contains users, they will also be available for importing into the
> current realm.

Reviewer: ```suggestion
NOTE: The Admin Console partial import can also import files created by the `export` CLI command. In other words, full exports created
by the CLI can be imported via Admin Console. If the file contains users, those users will also be available for importing into the
current realm.
```
